bitbucket-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+# Define the Docker image to use for all steps
+image: 
+  name: aaronmedinaaligent/shopify-cli-base:latest
+  username: $DOCKER_HUB_USERNAME
+  password: $DOCKER_HUB_PASSWORD
+
+definitions:
+  steps:
+    - step: &theme-check
+        name: Theme Check
+        services:
+          - docker
```

Comment:
Is this docker service needed here? Because of the way Bitbucket works, the build container might have lesser memory than intended because the services and build container share memory.

Reply:
Great catch! I initially included services: docker and caches: docker because I was running Docker commands in the first iteration of the pipeline. I overlooked removing them after making adjustments. Thanks for pointing that out, I'll clean it up accordingly!